apps/sync-asset-sg/tsconfig.app.json

```diff
@@ -3,7 +3,14 @@
   "compilerOptions": {
     "outDir": "../../dist/out-tsc",
     "module": "commonjs",
-    "types": ["node"]
+    "types": ["node"],
```

Comment:
Why do we need them explicitly here? They are in the baseconfig either way, no?

Reply:
Also, this module now targets `es2021`, whereas our baseconfig still has `es2015`